pkgs/tools/security/cve-bin-tool/default.nix

```diff
@@ -30,7 +30,42 @@
 , pip
 , testers
 , cve-bin-tool
+# pinned packaging
+, pyparsing
+, fetchPypi
+, buildPythonPackage
+, pretend
+, pythonOlder
 }:
+
+let
+  # pin packaging to < 22 until issue related to https://github.com/intel/cve-bin-tool/pull/2436 are resolved by upstrem
```

Comment:
Can you add a comment in which version of cve-bin-tool this pin can be removed?

Reply:
there is no upstream milestone, so the best we can do is watch the issue